lib/Backend/GlobOpt.cpp

```diff
@@ -17179,6 +17179,13 @@ GlobOpt::DoInlineArgsOpt(Func const * func)
 
 bool
 GlobOpt::IsSwitchOptEnabled(Func const * func)
+{
+    Assert(func->IsTopFunc());
+    return !PHASE_OFF(Js::SwitchOptPhase, func) && !func->IsSwitchOptDisabled() && func->DoGlobOpt();
+}
+
+bool
+GlobOpt::IsSwitchOptEnabledForIntTypeSpec(Func const * func)
 {
     Assert(func->IsTopFunc());
     return !PHASE_OFF(Js::SwitchOptPhase, func) && !func->IsSwitchOptDisabled() && !IsTypeSpecPhaseOff(func)
```

Comment:
Since this is a superset, can it call IsSwitchOptEnabled()?

Reply:
ahh...yes, it sure can!